service/healthz/k8s/service.go

```diff
@@ -0,0 +1,125 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/giantswarm/microendpoint/service/healthz"
+)
+
+const (
+	// Description describes which functionality this health check implements.
+	Description = "Ensure Kubernetes API availability."
+	// Name is the identifier of the health check. This can be used for emitting
+	// metrics.
+	Name = "k8s"
+	// SuccessMessage is the message returned in case the health check did not
+	// fail.
+	SuccessMessage = "all good"
```

Comment:
I just wonder if having space in success message may cause some scripting problems